kernels/volk/volk_32fc_convert_16ic.h

```diff
@@ -173,10 +180,9 @@ static inline void volk_32fc_convert_16ic_neon(lv_16sc_t* outputVector, const lv
 
     const float32x4_t min_val = vmovq_n_f32(min_val_f);
     const float32x4_t max_val = vmovq_n_f32(max_val_f);
-    float32x4_t half = vdupq_n_f32(0.5f);
-    float32x4_t ret1, ret2, a, b, sign, PlusHalf, Round;
+    float32x4_t ret1, ret2, a, b;
 
-    int32x4_t toint_a, toint_b;
+    int32x4_t toint_a={0,0,0,0}, toint_b={0,0,0,0};
```

Comment:
Could you split this into 2 lines?

Reply:
Done.

What do you think about doing a clang-reformat of VOLK like what was done for GNURadio? Especially in the `kernels/volk` subdirectory, the code formatting is quite erratic.

Reply:
I'm all for doing a clang-format for code consistency!

Reply:
We also need to get the license code header to be consistent.

Reply:
Off the top of my head I think GR contains a script for license header formatting. Should we open 2 issues for that? `Make license headers more consistent` and `Apply clang-format`?

Reply:
I'd say: yes! If you have time to do this, it would be great!